Lanchonete/Lanchonete/Model/Sandwich.swift

```diff
@@ -34,6 +34,7 @@ public enum SandwichType:  String, CaseIterable {
 
 struct Sandwich: Equatable {
     let type: SandwichType
+    //var ingredient: Ingredient
```

Comment:
Thank you, it will be removed in the next commit. Regarding your second question, the code itself is created to only support the menu sandwiches, so everything is somewhat hardcoded. It was a solution that was created to implement the simpler version of _Lanchonete_. This implementation will definitely change.